src/util/branch-name.ts

```diff
@@ -12,6 +12,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// cannot import from '..' - transpiled code references to RELEASE_PLEASE
+// at the script level are undefined, they are only defined inside function
+// or instance methods/properties.
+import {RELEASE_PLEASE} from '../constants';
```

Comment:
I really wanted to import from '..' but, even though typescript was happy and transpiled the code (`const __1 = require("..")`), the references to `__1.RELEASE_PLEASE` at the script level (where the `{COMPONENT,DEFAULT}_PATTERN` need to use them) were undefined. `__1.RELEASE_PLEASE` was only defined inside functions or instance methods/properties (I also tried static properties and it was undefined there too).

for some reason it works fine like this. transpiled code:
```
const constants_1 = require("../constants");
.
.
.
// happy as a clam
const DEFAULT_PATTERN = `^${constants_1.RELEASE_PLEASE}/branches/(?<branch>[^/]+)$`;
```

Reply:
I'm surprised by this behavior. For now, I suggest we stop exporting the constants in `index.ts`, like you suggest, to prevent hiccups.

Reply:
I tried really hard to find a reference on the internet but my searching skills were not up to the task. I'll update to remove the re-export from `index.ts`